### PR TITLE
Discord Webhook integration

### PR DESCRIPTION
**Pull request by RaidenvBlack**
Thats just how i think we could do it ^^ Let me know what you think

### CODE REVIEW

File: PushyFinder/Delivery/DncDeliveryDelivery.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    description = text,
+                    color = 16711680 // You can change the color here (e.g., 16711680 is red)
+                }
+            }
+            //content = title + " " + text
+        };
+
+        try
+        {
+
```
Comment (lostkagamine) [inline PushyFinder/Delivery/DncDeliveryDelivery.cs:39]: Unnecessary whitespace

File: PushyFinder/Delivery/DncDeliveryDelivery.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    private static async Task DeliverAsync(string title, string text)
+    {
+        // Replace 'YOUR_DISCORD_WEBHOOK_URL' with your actual Discord webhook URL
+        var discordWebhookUrl = Plugin.Configuration.DcHook;
+
+        // Constructing the payload for Discord webhook
+        var payload = new
+        {
+            username = "DcN",
```
Comment (lostkagamine) [inline PushyFinder/Delivery/DncDeliveryDelivery.cs]: Be descriptive.

File: PushyFinder/Impl/DutyListener.cs [hunk 3]
```diff
@@ -21,13 +21,28 @@ public static void Off()
 
     private static void OnDutyPop(ContentFinderCondition e)
     {
-        if (!Plugin.Configuration.EnableForDutyPops)
-            return;
+        if (Plugin.Configuration.EnablePushover)
+        {
+            if (!Plugin.Configuration.EnableForDutyPops)
```
Comment (lostkagamine) [inline PushyFinder/Impl/DutyListener.cs:26]: Moving this conditional is unnecessary.

File: PushyFinder/Impl/PartyListener.cs [hunk 4]
```diff
@@ -25,29 +25,63 @@ public static void Off()
...
-                                 $"{m.Name} (Lv{m.Level} {jobAbbr}) has left the party.");
+            PushoverDelivery.Deliver($"{m.PartyCount-1}/8: Party leave",
+                                     $"{m.Name} (Lv{m.Level} {jobAbbr}) has left the party.");
+        }
+
+        if (Plugin.Configuration.EnableDiscord)
+        {
+            if (!CharacterUtil.IsClientAfk()) return;
+        
+            var jobAbbr = LuminaDataUtil.GetJobAbbreviation(m.JobId);
```
Comment (lostkagamine) [inline PushyFinder/Impl/PartyListener.cs:81]: Don't repeat yourself!

File: PushyFinder/Plugin.cs [hunk 5]
```diff
@@ -13,7 +13,7 @@ namespace PushyFinder
     public sealed class Plugin : IDalamudPlugin
     {
         public string Name => "PushyFinder";
-        private const string CommandName = "/pushyfinder";
+        private const string CommandName = "/pf";
```
Comment (lostkagamine) [inline PushyFinder/Plugin.cs:16]: Don't change the command name!

File: PushyFinder/Plugin.cs [hunk 6]
```diff
@@ -45,7 +45,7 @@ public sealed class Plugin : IDalamudPlugin
 
             this.CommandManager.AddHandler(CommandName, new CommandInfo(OnCommand)
             {
-                HelpMessage = "Opens the configuration window."
+                HelpMessage = "Opens settings\n'pt' toggles Pushover whether it's enabled.\n'pon' enables the Pushover plugin\n'poff' disables the Pushover plugin.\n replace 'p' with 'd' for the Discord side"
```
Comment (lostkagamine) [inline PushyFinder/Plugin.cs:48]: No. These should be subcommands if they are even required; they aren't.

File: README.md [hunk 7]
```diff
@@ -2,7 +2,22 @@
 A Dalamud plugin to ping your phone via [Pushover](https://pushover.net) whenever a Party Finder event or duty pop happens with an AFK client.
 
 ## Usage
-Install the plugin, configure it using `/pushyfinder`, go AFK (look for the red icon) and enjoy.
+Install the plugin, configure it using `/pf`, go AFK (look for the red icon) and enjoy.
 
 You will need a valid [Pushover](https://pushover.net) license, application key and user key to use this.
 Support for other push notification services is coming soon, but for now, evaluate for yourself whether a one-time $5 purchase is worth it. We think it is.
+
```
Comment (lostkagamine) [inline README.md:9]: None of this needs to be explained in the readme

File: README.md [hunk 8]
```diff
@@ -2,7 +2,22 @@
 A Dalamud plugin to ping your phone via [Pushover](https://pushover.net) whenever a Party Finder event or duty pop happens with an AFK client.
 
 ## Usage
-Install the plugin, configure it using `/pushyfinder`, go AFK (look for the red icon) and enjoy.
+Install the plugin, configure it using `/pf`, go AFK (look for the red icon) and enjoy.
```
Comment (lostkagamine) [inline README.md:5]: Unnecessary change

**Review by lostkagamine — CHANGES_REQUESTED** [review]
There's a lot of things that need changing here.

File: PushyFinder/Plugin.cs (hunk 6 above)
Comment (RaidenvBlack) [reply PushyFinder/Plugin.cs:48]: They are required for the use in macros thats why i made them because the people i know that use that wanted to have an easy way to toggle these options

File: README.md (hunk 7 above)
Comment (RaidenvBlack) [reply README.md:9]: then i will need to explain it with a window ingame

File: PushyFinder/Impl/PartyListener.cs (hunk 4 above)
Comment (RaidenvBlack) [reply PushyFinder/Impl/PartyListener.cs:81]: i am not sure waht you mean

File: PushyFinder/Impl/DutyListener.cs (hunk 3 above)
Comment (RaidenvBlack) [reply PushyFinder/Impl/DutyListener.cs:26]: but it also not makes a problem to have it there or not ?

File: PushyFinder/Delivery/DncDeliveryDelivery.cs (hunk 1 above)
Comment (RaidenvBlack) [reply PushyFinder/Delivery/DncDeliveryDelivery.cs:39]: you mean at Line 36 ?

File: PushyFinder/Delivery/DncDeliveryDelivery.cs (hunk 2 above)
Comment (RaidenvBlack) [reply PushyFinder/Delivery/DncDeliveryDelivery.cs]: what do you mean ?

File: PushyFinder/Impl/PartyListener.cs (hunk 4 above)
Comment (lostkagamine) [reply PushyFinder/Impl/PartyListener.cs:81]: There's two instances of the LuminaDataUtil#GetJobAbbreviation call for no reason.

File: PushyFinder/Plugin.cs (hunk 5 above)
Comment (lostkagamine) [reply PushyFinder/Plugin.cs:16]: Don't break users' muscle memory.

File: PushyFinder/Plugin.cs (hunk 6 above)
Comment (lostkagamine) [reply PushyFinder/Plugin.cs:48]: Then you can do that in a separate PR, this goes beyond the scope of this one.

File: README.md (hunk 7 above)
Comment (lostkagamine) [reply README.md:9]: Just use a tooltip.

File: PushyFinder/Delivery/DncDeliveryDelivery.cs (hunk 1 above)
Comment (lostkagamine) [reply PushyFinder/Delivery/DncDeliveryDelivery.cs:39]: At line 39. Please follow the code style consistently.
